app/assets/stylesheets/_main.scss

```
@@ -307,3 +307,7 @@ html * {
 .italic {
   font-style: italic;
 }
+
+.govuk-table__cell.bottom-border-none {
+  border-bottom: none;
+}
```

This is a pretty minor point, but I'm not a huge fan of having to add a class to the cells based on the contents of the next row. How about using `:has()` to detect whether it's followed by a particular element?

```suggestion
.govuk-table__row:has(+ .audit) .govuk-table__cell {
  border-bottom: none;
}
```

And then instead of adding the `bottom-border-none` class on L3 of `row_component`, we could add an `audit` class (or whatever name) on L14 instead, inside the `if audit` conditional.

The downside is this might be a little more complex to read and browser support isn't 100% yet, so feel free to say no to this.

---

Yeah I agree with where you're coming from. I'm actually thinking whether I just do:

```
.bottom-border-none {
  border-bottom: none;
}
```
since this css rule doesn't necessarily have to follow a `govuk-table__cell` and should be able to be reused anywhere